app/controllers/Checkout.scala

```diff
@@ -300,30 +305,40 @@ class Checkout(fBackendFactory: TouchpointBackends, commonActions: CommonActions
         _ + _
       }
 
-      val acquisitionData = request.session.get("acquisitionData")
-      if (acquisitionData.isEmpty) {
-        logger.warn(s"No acquisitionData in session")
+      submitAcquisitionEvent(request, subscribeRequest).value.map { _ =>
+        logger.info(s"User successfully became subscriber:\n\tUser: SF=${r.salesforceMember.salesforceContactId}\n\tPlan: ${subscribeRequest.productData.fold(_.plan, _.plan).name}\n\tSubscription: ${r.subscribeResult.subscriptionName}")
+        Ok(Json.obj("redirect" -> routes.Checkout.thankYou().url)).withSession(session)
       }
+    }
 
-      val promotion = subscribeRequest.genericData.promoCode.map(_.get).flatMap(code => tpBackend.promoService.findPromotion(NormalisedPromoCode.safeFromString(code)))
-      val clientBrowserInfo = ClientBrowserInfo(
-        request.cookies.get("_ga").map(_.value).getOrElse(UUID.randomUUID().toString), //Get GA client id from cookie
-        request.headers.get("user-agent"),
-        request.remoteAddress
-      )
+    def submitAcquisitionEvent(request: Request[AnyContent], subscribeRequest: SubscribeRequest): EitherT[Future, Unit, AcquisitionSubmission] = {
+      val testUser = TestUsers.isTestUser(PreSigninTestCookie, request.cookies)(request)
+      if(testUser.isEmpty) {
+        val acquisitionData = request.session.get("acquisitionData")
+        if (acquisitionData.isEmpty) {
+          logger.warn(s"No acquisitionData in session")
+        }
 
-      //This service is mocked unless it's running in PROD, change to test acquisition events are working
-      AcquisitionService(isTestService = tpBackend.environmentName != "PROD")
-        .submit(SubscriptionAcquisitionComponents(subscribeRequest, promotion, acquisitionData, clientBrowserInfo))
-        .leftMap(
-          err => logger.warn(s"Error submitting acquisition data. $err")
+        val promotion = subscribeRequest.genericData.promoCode.map(_.get).flatMap(code => tpBackend.promoService.findPromotion(NormalisedPromoCode.safeFromString(code)))
+        val clientBrowserInfo = ClientBrowserInfo(
+          request.cookies.get("_ga").map(_.value).getOrElse(UUID.randomUUID().toString), //Get GA client id from cookie
+          request.headers.get("user-agent"),
+          request.remoteAddress
         )
 
-      logger.info(s"User successfully became subscriber:\n\tUser: SF=${r.salesforceMember.salesforceContactId}\n\tPlan: ${subscribeRequest.productData.fold(_.plan, _.plan).name}\n\tSubscription: ${r.subscribeResult.subscriptionName}")
-      Ok(Json.obj("redirect" -> routes.Checkout.thankYou().url)).withSession(session)
+        //This service is mocked unless it's running in PROD, change to test acquisition events are working
+        AcquisitionService(isTestService = tpBackend.environmentName != "PROD")
```

Comment:
I had always assumed that we'd create a mock acquisition service here as the environment name = UAT for test users?